src/assessments/seo/subheadingsKeywordAssessment.js

```diff
@@ -92,17 +114,49 @@ class SubHeadingsKeywordAssessment extends Assessment {
 	 * @returns {string} The translated string.
 	 */
 	translateScore( score, subHeadings, i18n ) {
-		if ( score === this._config.scores.multipleMatches || score === this._config.scores.oneMatch ) {
+		let minNumberOfSubheadings = subHeadings.count * this._config.lowerBoundary;
```

Comment:
These calculations are also done inside the calculateScore function. Can you make minNumberOfSubheadings and maxNumberOfSubheadings globally available within this file?